lib/discourse_plugin_registry.rb

```diff
@@ -68,6 +68,7 @@ def self.define_filtered_register(register_name)
   define_register :vendored_core_pretty_text, Set
   define_register :seedfu_filter, Set
   define_register :demon_processes, Set
+  define_register :group_filter_for_search, Hash
```

Comment:
```suggestion
  define_register :groups_callback_for_users_search_controller_action, Hash
```

Perhaps this will make it clearly what it is used for?